### PR TITLE
ADD Signal sample for mono photon

### PR DESCRIPTION
**Pull request by smdogra**
ADD Signal sample for mono photon  files for UL samples

### CODE REVIEW

**Comment by Saptaparna** [comment]
Hi @smdogra could you please commit files for only one scenario with a script that will generate the various configs? The review will make the review go faster.

**Comment by smdogra** [comment]
Hi @Saptaparna I have kept one single file with createcards_mphotoncp6tune.csh script which create other scenarios. 
Thanks

File: genfragments/ThirteenTeV/ADD/createcards_mphotoncp6tune.csh [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+import FWCore.ParameterSet.Config as cms
+
+
+
+from Configuration.Generator.Pythia8CommonSettings_cfi import *
+from Configuration.Generator.MCTunes2017.PythiaCP5Settings_cfi import *
+from Configuration.Generator.PSweightsPythia.PythiaPSweightsSettings_cfi import *
+
+
+generator = cms.EDFilter("Pythia8GeneratorFilter",
```
Comment (menglu21) [inline genfragments/ThirteenTeV/ADD/createcards_mphotoncp6tune.csh:19]: could you please change it to concurrent mode one

File: genfragments/ThirteenTeV/ADD/ADDmonoPhoton_MD_1000_d_3_TuneCP5_13TeV_pythia8_cfi.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+import FWCore.ParameterSet.Config as cms
+
+
+
+from Configuration.Generator.Pythia8CommonSettings_cfi import *
+from Configuration.Generator.MCTunes2017.PythiaCP5Settings_cfi import *
+from Configuration.Generator.PSweightsPythia.PythiaPSweightsSettings_cfi import *
+
+
+generator = cms.EDFilter("Pythia8GeneratorFilter",
```
Comment (menglu21) [inline genfragments/ThirteenTeV/ADD/ADDmonoPhoton_MD_1000_d_3_TuneCP5_13TeV_pythia8_cfi.py:11]: could you please change it to concurrent mode one

Comment (smdogra) [reply genfragments/ThirteenTeV/ADD/ADDmonoPhoton_MD_1000_d_3_TuneCP5_13TeV_pythia8_cfi.py]: Done

File: genfragments/ThirteenTeV/ADD/createcards_mphotoncp6tune.csh (hunk 1 above)
Comment (smdogra) [reply genfragments/ThirteenTeV/ADD/createcards_mphotoncp6tune.csh]: @menglu21 Done

**Pull request merged**